### ISSUE TITLE
"parse error: Invalid literal at line 1, column 3" when Folder ID contains a space/blank

### ISSUE THREAD

**Issue by thomasmerz** [serl/syncthing-quick-status#8]
@serl 
## Problem
When a **Folder ID contains a space/blank** your script runs into the following error:
```
…
Docker: parse error: Invalid literal at line 1, column 3
parse error: Invalid literal at line 1, column 3

Volumes: parse error: Invalid literal at line 1, column 3
parse error: Invalid literal at line 1, column 3
…
```
vs.
<img width="547" alt="image" src="https://user-images.githubusercontent.com/18568381/153466247-173f10c7-735e-4dda-bb9c-19d033ec7f52.png">

## Reproduce
Problem can be reproduced by placing a file into any syncthing-folder that the user running  `syncthing@user.service` has no permissions to read.

## Expectation
Any user will expect that your script can handle also work this situation.

## Optional
Check for other "special" characters that might break parsing the output.

**Comment by serl** [comment]
Hello,

Thanks for the heads-up! That should be fixed now.


...Anyway, your "Reproduce" part is clearly for another issue, and I'd like to see how you create folder ids with spaces. I guess you're doing something not supported by Syncthing itself 🙈 😆 

<img width="883" alt="image" src="https://user-images.githubusercontent.com/2742655/153767649-47ef1472-4379-4448-afe7-251b4dd3f331.png">

**Comment by thomasmerz** [comment]
Well, it looks like it was possible to create folder ids with spaces in some earlier versions, because I have some 😆 

<img width="306" alt="grafik" src="https://user-images.githubusercontent.com/18568381/153775153-e4893cae-75ec-41a6-8c5d-7f8ac6492534.png">


Reproduce: This looks like I've been doing to much in parallel, mixing up my own issues 🤣
